WordPress/build.gradle

```diff
@@ -36,12 +36,16 @@ repositories {
             includeGroup "org.wordpress.aztec"
             includeGroup "org.wordpress.fluxc"
             includeGroup "org.wordpress.wellsql"
-            includeGroup "org.wordpress-mobile"
-            includeGroup "org.wordpress-mobile.gutenberg-mobile"
-            includeGroupByRegex "org.wordpress-mobile.react-native-libraries.*"
+            includeGroup "org.wordpress.gutenberg-mobile"
+            includeGroupByRegex "org.wordpress.react-native-libraries.*"
             includeGroup "com.automattic"
             includeGroup "com.automattic.stories"
             includeGroup "com.automattic.tracks"
+            // 'org.wordpress-mobile' group is deprecated. It's kept for now for smoother transition
+            // but it should be removed soon (within couple weeks)
```

Comment:
We could remove `org.wordpress-mobile` once version 24.1 is code frozen and we generate an alpha version of Gutenberg Mobile to include changes from https://github.com/WordPress/gutenberg/pull/58017.